packages/ui-tailwind/src/display/Icon/Icon.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import NotFoundIcon from "@/assets/icons/404.svg";
```

Comment:
It looks like you haven't used this icon inside the component

Reply:
Okay... That's a mistake here, I think I will keep this here for now, I should be configuring webpack to load svg in the next pr (assuming I'm understanding this correctly)